### PR TITLE
common(tile): read-more/read-less button added

### PR DESCRIPTION
**Pull request by damdlani**

### CODE REVIEW

File: src/common/Tile/index.js [hunk 1]
```diff
@@ -114,10 +120,13 @@ export const Tile = ({
     {(tileView === "detail") ?
         <Container description tileView={tileView}>
           <Description>
-            {description}
+              {description.length < 500  ? description : 
+              (description.length >= 500 && !readMore) ? <>{description.substring(0, 501)}...<ReadMore onClick={toggleReadMore}>read more</ReadMore></> :
+              (description.length >= 500 && readMore) ? <>{description}<ReadMore onClick={toggleReadMore}>read less</ReadMore></> : ""}
```
Comment (anetaszynal) [inline src/common/Tile/index.js:125]: Do you need this condition : "(description.length >= 500 && readMore) ?" ?

File: src/common/Tile/styled.js [hunk 2]
```diff
@@ -177,7 +177,7 @@ export const Header = styled.h2`
 
     ${({ tileView }) => tileView === "list" && css`
         font-weight: 500;
-        font-size: 22px;
+        font-size: 20px;
```
Comment (anetaszynal) [inline src/common/Tile/styled.js:180]: Nice that you correct this :)

File: src/common/Tile/styled.js [hunk 3]
```diff
@@ -477,4 +477,24 @@ export const NoImagePlug = styled.div`
...
     width: 25%;
+`;
+
+export const ReadMore = styled.button`
+    color: ${({ theme }) => theme.color.darkerGrey};
+    background-color: transparent;
+    border: none;
+    font-size: 0.75em;
+    padding: 0;
+
```
Comment (anetaszynal) [inline src/common/Tile/styled.js:488]: What do you think to add transition here?

**Review by anetaszynal — APPROVED** [review]

File: src/common/Tile/styled.js (hunk 3 above)
Comment (damdlani) [reply src/common/Tile/styled.js:488]: Good point

**Pull request merged**